exchange/utils.go

```diff
@@ -396,17 +399,28 @@ func splitImps(imps []openrtb2.Imp) (map[string][]openrtb2.Imp, error) {
 }
 
 func createSanitizedImpExt(impExt, impExtPrebid map[string]json.RawMessage) (map[string]json.RawMessage, error) {
-	sanitizedImpExt := make(map[string]json.RawMessage, 3)
+	sanitizedImpExt := make(map[string]json.RawMessage, 8)
+	sanitizedImpPrebidExt := make(map[string]json.RawMessage, 8)
```

Comment:
Shouldn't the size of these be 6 and 2 respectively instead of 8 and 8?

Reply:
Yes